### ISSUE TITLE
aws_autoscaling_group conflicts with tags

### ISSUE THREAD

**Issue by cknowles** [env0/terratag#93]
Using env0 with https://registry.terraform.io/modules/terraform-aws-modules/autoscaling/ or https://registry.terraform.io/modules/cloudposse/ec2-autoscale-group/ receives the below error (just use any basic documented example). It doesn't seem to matter whether it's Terraform 0.15.x or 1.0.x. Judging from [the code](https://github.com/env0/terratag/blob/f8f06a1281c58945f7fca0ffd05a7efe8729f925/tagging/aws.go#L8-L10) it seems like it's inserting `tag` attributes which then conflict with the `tags` used from the modules. I didn't spot any way to turn off the tagging in env0 meaning it's a blocker.

> │ Error: Conflicting configuration arguments
> |
> │ "tags": conflicts with tag

**Comment by avnerenv0** [comment]
Hi @cknowles 

We'll look into the issue with these resources.

Regarding - "I didn't spot any way to turn off the tagging in env0" - that's not really related to this repository - but tagging in env0 is turned on/off according to each Project's settings. You can find more about this here - https://docs.env0.com/docs/cost-monitoring

**Comment by cknowles** [comment]
@avnerenv0 cool, thanks. That's unblocked it for now, had misread the text there thinking it was only querying the APIs using those credentials.

**Comment by ezraroi** [comment]
any updates regarding this?

**Comment by yaronya** [comment]
Duplicate of #101
